internal/core/graphics/image.rs

```diff
@@ -686,6 +687,65 @@ impl Image {
     }
 }
 
+/// This enum describes the origin to use when rendering a borrowed OpenGL texture.
+/// Use this with [BorrowedOpenGLTextureBuilder::origin].
+#[derive(Copy, Clone, Debug, PartialEq, Default)]
+#[repr(C)]
+pub enum BorrowedOpenGLTextureOrigin {
+    /// The top-left of the texture is the top-left of the texture drawn on the screen.
+    #[default]
+    TopLeft,
+    /// The bottom-left of the texture is the top-left of the texture draw on the screen,
+    /// flipping it vertically.
+    BottomLeft,
+}
+
+/// Factory to create [`slint::Image`](crate::graphics::Image) from an existing OpenGL texture.
+///
+/// Methods can be chained on it in order to configure it.
+///
+///  * `origin`: Change the texture's origin when rendering (defaullt: TopLeft).
+///
```

Comment:
Could there be a sample example of usage?

Reply:
I changed the texture import example to use the builder (that's how I tested it :), but for the flipping itself: I'd say it's a bit of a legacy feature. You integrate some third-party rendered content, find out that they use the original GL coordinate system, flip this switch, and that's it. Normally changing the origin should not be needed.

Reply:
Ahh, yes, I can do that :)